pikaday.js

```diff
@@ -425,7 +427,13 @@
                 opts.i18n.months[i] + '</option>');
         }
 
-        monthHtml = '<div class="pika-label">' + opts.i18n.months[month] + '<select class="pika-select pika-select-month" tabindex="-1">' + arr.join('') + '</select></div>';
+        monthInnerHtml = opts.i18n.months[month] + '<select class="pika-select pika-select-month" tabindex="-1">' + arr.join('');
+
+        if (instance.el.querySelector('div#' + randId + ' .pika-label-month')) {
+            instance.el.querySelector('div#' + randId + ' .pika-label-month').innerHTML = monthInnerHtml;
+        } else {
+            monthHtml = '<div class="pika-label pika-label-month">' + monthInnerHtml + '</select></div>';
```

Comment:
do we need `pika-label-month` class here?

Reply:
The usage of `pika-label-month` is useful when we update it HTML (same with `pika-label-year`)

- `if (instance.el.querySelector('div#' + randId + ' .pika-label-month'))` 
- `if (instance.el.querySelector('div#' + randId + ' .pika-label-year'))`